pkg/controllers/vrep/replication_reconciler.go

```diff
@@ -150,7 +159,7 @@ func (r *ReplicationReconciler) fetchVdbs(ctx context.Context) (res ctrl.Result,
 // makeDispatcher will create a Dispatcher object based on the feature flags set.
 func (r *ReplicationReconciler) makeDispatcher() error {
 	if vmeta.UseVClusterOps(r.SourceInfo.Vdb.Annotations) {
-		r.dispatcher = vadmin.MakeVClusterOps(r.Log, r.SourceInfo.Vdb,
+		r.dispatcher = vadmin.MakeVClusterOpsWithTarget(r.Log, r.SourceInfo.Vdb, r.TargetInfo.Vdb,
```

Comment:
We need to check if this is async replication, if not, we should use "MakeVClusterOps"